includes/multi-currency/Analytics.php

```diff
@@ -17,6 +17,8 @@
  * Class that contains Multi-Currency related support for WooCommerce analytics.
  */
 class Analytics {
+	const FILTER_PREFIX = 'wcpay_multi_currency_';
```

Comment:
Nitpick, but what do you think about moving this const to the main `MultiCurrency` class? I noticed we also have the same `FILTER_PREFIX` defined in the `Compatibility` class so potentially we could just define the constant in the main `MultiCurrency` class and then use it in both places (Analytics and Compatibility) with `MultiCurrency::FILTER_PREFIX`.

If you think it's more explicit/clear to define it here though, that's fine too.

Reply:
I had been thinking about the constant because it's annoying to have the repetitive code. It didn't click to me that it would be available as `MultiCurrency::FILTER_PREFIX` 🤦  . Updated in https://github.com/Automattic/woocommerce-payments/pull/3286/commits/55b63fbd641ad04bbd993b6359df7e22b2d74b72

Reply:
Looks great, thanks for making that change!